### DISCUSSION TITLE
zh/guide/advanced/balance

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#63] [Announcements]
# zh/guide/advanced/balance

AList v3的文档

https://alist.nn.ci/zh/guide/advanced/balance.html

**Comment by shaojs321** [comment]
能负载均衡上传吗

**Comment by RainbowXie** [comment]
没啥卵用的功能，上传不同步，所以一上传目录就乱了，然后就不正常了。

**Reply by Winyer** [reply]
和我想的负载均衡不一样，我以为的是，比如AB两个存储，A中的文件新增或删除，B也随之新增或者删除，比如A创建了个文件夹名为1，在里面放了个文件1.mp4，B也会创建文件夹1并下载A的1.mp4上传进去，而A删除了文件夹2时，B也会删除文件夹2这样😥

**Comment by 100yuan** [comment]
应该负载均衡传输前，提前对每一个负载盘做带宽测试，根据带宽比例分配流量权重，这样负载才能最优化。

**Comment by Winyer** [comment]
和我想的负载均衡不一样，我以为的是，比如AB两个存储，A中的文件新增或删除，B也随之新增或者删除，比如A创建了个文件夹名为1，在里面放了个文件1.mp4，B也会创建文件夹1并下载A的1.mp4上传进去，而A删除了文件夹2时，B也会删除文件夹2这样😥

**Reply by RainbowXie** [reply]
按道理应该这么做，但是没有这么做。

**Comment by Harrison3-1122** [comment]
能不能加个同步功能

**Comment by mixiaobu** [comment]
完全一致也报错用不了

**Comment by xiaoyi0805** [comment]
这个就是能不能实现，请求负载，云盘请求次数过多的话就封号。能两个同样目录结构的云盘，当一个请求次数超过某个值的时候，就请求另外一个。这样就不容易封号了
